dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -27,7 +27,7 @@
 .script_text
   = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
 
-- if [ContractMatch, Match, Multi, TextMatch, External].include? @level.class
+- if [ContractMatch, Match, Multi, TextMatch, External, LevelGroup].include? @level.class
```

Comment:
Thanks both, I've made markdown editing now optional, and taken LevelGroup out of the list for now.  I've also renamed the .js file to levelbuilder_markdown.

It would be neat for levels to have a markdown_enabled? method but am putting this on my to-do list while circling to other assessment work right now.